github_api.py

```diff
@@ -0,0 +1,81 @@
+import json
+import re
+
+import requests
+from flask import abort
+
+import config
+import routes.overview_page
+
+
+pattern = r"(?:https?://)?(?:www\.)?github\.com/([\w-]+)/([\w-]+)/?"
+
+
+def get_github_urls(links):
+    github_urls = set()
+    for category in links["categories"]:
+        for repo in category["category_repos"]:
+            for link in repo["links"]:
+                if re.match(pattern, link["link_value"]):
+                    github_urls.add(link["link_value"])
+
+    return sorted(list(github_urls), key=lambda x: x.split("/")[-1])
+
+
+def get_open_pull_request():
+    # Get list of GitHub repos from Overview page
+    services_links = routes.overview_page.get_services_links()
+    github_urls = get_github_urls(services_links)
+
+    pull_requests = {}
+    gh_token = config.GITHUB_TOKEN
```

Comment:
What will happen if the token is not defined and therefore `None` here?

Reply:
then 401 is raised (L67)